operations/availabilityblocks.md

```diff
@@ -118,6 +118,7 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
             "FirstTimeUnitStartUtc": "2021-10-14T00:00:00Z",
             "LastTimeUnitStartUtc": "2021-10-17T00:00:00Z",
             "ReleasedUtc": "2021-10-13T00:00:00Z",
+            "RollingReleaseOffset": "P-3DT4H",
```

Comment:
If the two release fields are optional and "mutually exclusive", doesn't that mean that only one of them will be present in the JSON? The example now shows both, which violates that rule.